setup.py

```diff
@@ -80,6 +80,7 @@
 _deps = [
     "Pillow",  # keep the PIL.Image.Resampling deprecation away
     "accelerate>=0.11.0",
+    "compel",
```

Comment:
```suggestion
    "compel==0.1.8",
```
My personal rule of thumb is that packages which aren't used by many people or by adhoc maintainers should be version pinned. They aren't as common targets for supply chain attacks but the maintainers are less likely to have good security posture.

Pip does support hash pinning but it looks like it requires opt in on install which I assume most people don't do. Pypi does not allow re-uploads of already released versions so version pinning is sufficient imo. https://flawed.net.nz/2021/02/02/PyPI-Security-State/

I'm not an expert on this, and I usually run small dependency stuff like this by an internal appsec team or something similar.

Reply:
yes you're 100% right - compel 0.1.8 was only just released ;-)